components/server/src/workspace/workspace-starter.ts

```diff
@@ -850,25 +850,37 @@ export class WorkspaceStarter {
                 );
             }
 
-            if (
-                await getExperimentsClientForBackend().getValueAsync("protected_secrets", false, {
-                    user,
-                    teams: userTeams,
-                    billingTier,
-                })
-            ) {
-                // We roll out the protected secrets feature using a ConfigCat feature flag, to ensure
-                // a smooth, gradual roll out without breaking users.
-                featureFlags = featureFlags.concat(["protected_secrets"]);
```

Comment:
Up to my knowledge, this feature flag has no effect anymore for more 1 week now. (itÄs a default workspace feature flag since then).